core/src/main/java/org/infinispan/protostream/annotations/impl/types/XElement.java

```diff
@@ -47,5 +47,5 @@ default boolean isPrivate() {
     *
     * @return the documentation or {@code null} if no doc or empty doc present
     */
-   String getProtoDocs();
+   String getDocumentation();
```

Comment:
Probably worth mentioning in the JavaDoc that "custom" annotations are included as well.